.github/workflows/build.yml

```
@@ -0,0 +1,35 @@
+name: Build
+
+on:
+  pull_request:
+    branches: [ main, master, develop ]
+
+env:
+  CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: 1
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+
+      - name: Rust cache
+        uses: Swatinem/rust-cache@v2
+        with:
+          shared-key: "sdk-cache"
+          
+      - name: Run cargo build
+        uses: actions-rs/cargo@v1
```

We should use dtolnay toolchain since action-rs is archived project
```suggestion
        uses: dtolnay/rust-toolchain@stable
```